pkg/controller/nodelifecycle/metrics.go

```diff
@@ -79,6 +79,25 @@ var (
 		},
 		[]string{"zone"},
 	)
+
+	updateNodeHealthDuration = metrics.NewHistogram(
+		&metrics.HistogramOpts{
+			Subsystem:      nodeControllerSubsystem,
+			Name:           "update_node_health_duration_seconds",
+			Help:           "Duration in seconds for NodeController to update the health of a single node.",
+			Buckets:        metrics.ExponentialBuckets(0.001, 2, 16), // 1ms -> ~30s
+			StabilityLevel: metrics.ALPHA,
+		},
+	)
+	monitorNodeHealthDuration = metrics.NewHistogram(
+		&metrics.HistogramOpts{
+			Subsystem:      nodeControllerSubsystem,
+			Name:           "update_nodes_health_duration_seconds",
+			Help:           "Duration in seconds for NodeController to update the health of all nodes.",
+			Buckets:        metrics.ExponentialBuckets(0.01, 2, 16), // 10ms -> ~5m
```

Comment:
I don't think we need 16 buckets. I'm also not convinced we need that big granularity - probably having metrics.ExponentialBuckets(0.001, 6) is good enough...